students/pinstripezebra/mailroomP3

```diff
@@ -0,0 +1,128 @@
+"""
+Created on Sat Feb 10 08:23:07 2018
+
+@author: seelc
+"""
+import datetime
+
+#Dictionary containing all donor data
+donationHistory = {'William Gates, III':[10312.3, 3],
+                   'Mark Zuckerberg':[31245.24, 3],
+                   'Jeff Bezos':[9432.01, 2],
+                   'Paul Alan':[7244.35, 2],
+                   'Elon Musk':[13148.94, 1],
+        }
+
+def main():
+    #promting the user to choose a main action
+    userAction = ""
+    while userAction !="4":
+        userAction = input( "Choose an action: \n1- Send a thank you\n2-Create a Report\n3-Send a thank you to everyone\n4-Quit\n")
+        #Attempting to throw an exception if userAction isnt 1, 2, or 3
+        try:
+            assert userAction =="1" or "2" or "3" or "4"
```

Comment:
you could probably clean this up better with a:
```
while user_action not in ("1", "2", "3", "4"):
    continue
```
this will make it go to the top of your `while` loop on line 19 and start over.  Or you could just add the `continue` to your exception catch.